scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatOps.scala

```diff
@@ -2081,15 +2086,20 @@ class FormatOps(
     }
 
     object BlockImpl extends Impl {
+
```

Comment:
this empty line makes it inconsistent with other Impl implementations?

Reply:
Ach sorry, that's left over from debugging. Got thrown off by the checks not running and haven't checked for silly changes

Reply:
Removef